claripy/ast/base.py

```diff
@@ -975,7 +966,7 @@ def __getattr__(self, a):
             return self
 
 def simplify(e):
-    if isinstance(e, Base) and e.op == 'I':
+    if type(e) is BV and e.op == 'BVV':
```

Comment:
I think this one should check all of leaf_operations. There's not much we can do to simplify an `FPV` or a `BoolV`, (or, I guess sometime soon, a StringV), either. Back when this was checking for `I`, it would have matched any immediate, including booleans and floating points.